### PR TITLE
Allow fetching EBS volumes

### PR DESCRIPTION
**Pull request by bombergio**
With this PR adding new config argument `ec2.ebs_volumes` to fetch EC2 EBS volumes.

### CODE REVIEW

File: resources/ec2_ebs_volumes.go [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+				Name: "encrypted",
+				Type: schema.TypeBool,
+			},
+			{
+				Name: "fast_restored",
+				Type: schema.TypeBool,
+			},
+			{
+				Name: "iops",
+				Type: schema.TypeInt,
```
Comment (James-Quigley) [inline resources/ec2_ebs_volumes.go:52]:
```suggestion
				Type: schema.TypeBigInt,
```

File: resources/ec2_ebs_volumes.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+				Name: "multi_attach_enabled",
+				Type: schema.TypeBool,
+			},
+			{
+				Name: "outpost_arn",
+				Type: schema.TypeString,
+			},
+			{
+				Name: "size",
+				Type: schema.TypeInt,
```
Comment (James-Quigley) [inline resources/ec2_ebs_volumes.go:68]:
```suggestion
				Type: schema.TypeBigInt,
```

**Review by James-Quigley — CHANGES_REQUESTED** [review]
A few comments:
- I think types that are int64 should correspond to BigInt?
- I think this is the right schema from the SDK right? https://pkg.go.dev/github.com/aws/aws-sdk-go-v2@v0.24.0/service/ec2#Volume If so, I don't see `throughput` in the schema, and it seems like `Attachments` were omitted
- Need to run the generator for the mocks to update the ec2 service mock, and then need to implement the mock test for ec2 Volumes. Follow the instructions in https://github.com/cloudquery/cq-provider-aws/blob/main/adding_a_new_resource.md

**Comment by bombergio** [comment]
You are referencing outdated EC2 SDK - v 0.24.0, while I was using v 1.2.0 (and the same version is used in the [module](https://github.com/cloudquery/cq-provider-aws/blob/main/go.mod#L15) )
As you can see in the [EC2 v1.2.0](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/ec2@v1.2.0/types#Volume), all integer fields are of type `int32` and there is a field name `throughput`.
Thanks for pointing missing `Attachments` and missing mocks, I'll add them now.

File: resources/ec2_ebs_volumes.go (hunk 1 above)
Comment (bombergio) [reply resources/ec2_ebs_volumes.go:53]: Type is `int32`, so no need to change it to the `TypeBigInt`

File: resources/ec2_ebs_volumes.go (hunk 2 above)
Comment (bombergio) [reply resources/ec2_ebs_volumes.go:69]: Type is `int32`, so no need to change it to the `TypeBigInt`

**Comment by yevgenypats** [comment]
@bombergio Thanks! Merging.

**Pull request merged**